index.bs

```diff
@@ -3967,6 +3976,38 @@ Note: The {{UserVerificationRequirement}} enumeration is deliberately not refere
         interest of minimizing disruption to the user interaction flow).
 </div>
 
+### Client Capability Enumeration (enum <dfn enum>ClientCapability</dfn>) ### {#enum-clientCapability}
+
+<xmp class="idl">
+    enum ClientCapability {
+        "user-verifying-platform-authenticator",
+        "hybrid-transport",
```

Comment:
Would this be different from [`navigator.bluetooth.getAvailability()`](https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetooth-getavailability)? We can always recommend people call that method. And that way, we would not have to reimplement any privacy guards (see step 2, if the user denied permission previously the ua can return false).

As discussed in our meeting, user agents should not indicate bluetooth powered-on state to rps silently.

Reply:
(this got addressed below)